docs/source/index.rst

```diff
@@ -64,7 +64,7 @@ Algorithm and visualization codes for **latitude-longitude contour maps**,
 **polar contour maps**, the accompanying **summarizing table** and **Taylor diagram plots**,  **pressure-latitude zonal mean contour plots**,
 **zonal mean line plots**, **pressure-longitude meridional mean contour plots**, **area mean time series plots**, and **Cloud Top Height-Tau** joint histograms
 from COSP cloud simulator output. Plots can be created for annual
-and seasonal climatologies, and monthly mean time series.
+and seasonal climatologies, and monthly mean time series. **Diurnal cycle phase plot**, **Streamflow evaluation**, **ARM diags**, and **TC analysis** are implemented in newer releases in addition to the core sets aformentioned.
```

Comment:
This might flow more smoothly: 

In additional to the core sets, **Diurnal cycle phase plot**, **Streamflow evaluation**, **ARM diags**, and **TC analysis** are all implemented in v2.5.0.

Reply:
Fixed